### DISCUSSION TITLE
Welcome to chrome-passwordmaker Discussions!

### DISCUSSION THREAD

**Discussion by heavensrevenge** [passwordmaker/chrome-passwordmaker#191] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by heavensrevenge** [comment]
By request of @jeremy-harnois, Discussions have been enabled!  I have been away for a few years but am back to help maintain and add new features etc.  I have done the migration to manifest version 3, and hope the new Chrome Web Store @GitTom author will push a new version soon.

**Comment by GitTom** [comment]
Yes, I was concerned that the new version required a new permission - the 'scripting' permission - and was concerned that we should provide an explanation for why we needed it.   @heavensrevenge has explained to me that this permission is a silent permission: it won't require user acknowledgement so there shouldn't be a problem.

I will submit the updated version in the next couple days.  I have no idea how long it takes for Google to process before it gets published.

I'm glad that the extension will now be manifest v3.  Thanks!
